core/components/css-utilities/Display/Display.story.tsx

```diff
@@ -75,28 +69,155 @@ export const display = () => {
           />
         </Card>
       </div>
-      <Heading size="m">Examples</Heading>
-      <Paragraph>Here are some representative examples of these classes:</Paragraph>
-      <br />
-      <div className="d-inline p-2 Utilities-example">d-inline</div>
-      <div className="d-inline p-2 bg-dark Utilities-example Utilities-text--color">d-inline</div>
-      <br />
-      &nbsp;
-      <div className="DocPage-codeBlock pb-5 pt-5 pl-5">
-        {'<div className="d-inline p-2">d-inline</div>'}
-        <br />
-        {'<div className="d-inline p-2">d-inline</div>'}
-        <br />
+
+      {/* Examples section */}
+      <Heading size="m" className="mt-8">
+        Examples
+      </Heading>
+      <Paragraph appearance="default">Here are examples of these display utilities in action:</Paragraph>
+
+      {/* d-inline example */}
+      <div className="mt-8 mb-8">
+        <Heading size="s" className="mb-3">
+          Inline Display
+        </Heading>
+        <div className="mb-4">
+          <div className="d-inline bg-stone-lightest border border-2 border-stone-light p-2 px-4 mr-2 mb-2 align-middle lh-8">
+            d-inline
+          </div>
+          <div className="d-inline bg-jal-lightest border border-2 border-jal-light p-2 px-4 mr-2 mb-2 align-middle lh-8">
+            d-inline
+          </div>
+          <div className="d-inline bg-neem-lightest border border-2 border-neem-light p-2 px-4 mr-2 mb-2 align-middle lh-8">
+            d-inline
+          </div>
+          <div className="d-inline bg-tawak-lightest border border-2 border-tawak-light p-2 px-4 mr-2 mb-2 align-middle lh-8">
+            d-inline
+          </div>
+          <div className="d-inline bg-mirch-lightest border border-2 border-mirch-light p-2 px-4 mr-2 mb-2 align-middle lh-8">
+            d-inline
+          </div>
+        </div>
+        <div className="mb-2 text-muted color-text-subtle font-size-xs">
+          Elements with d-inline display as inline elements, appearing side by side and taking up only as much width as
+          necessary. They will wrap to new lines when they reach the container's edge.
+        </div>
+        <div className="bg-stone-lightest p-3 rounded-10 font-monospace font-size-xs">
```

Comment:
please remove all these unused classes which does not even exist in our design system like:

- text-muted
- bg-stone-lightest
- border-2
- border-stone-light
- lh-8
- bg-jal-lightest
- border-jal-light
- bg-neem-lightest
- border-neem-light
- bg-tawak-lightest
- border-tawak-light
- bg-mirch-lightest
- border-mirch-light
- color-text-subtle
- bg-stone-lightest

And also it does not make any sense to use `p-2 px-4` together.